packages/story-editor/src/components/mediaRecording/footer.js

```diff
@@ -318,12 +326,18 @@ function Footer({ captureImage, videoRef }) {
       resource: objectPick(resource, Object.keys(BackgroundAudioPropTypeShape)),
     };
 
+    const backgroundElement = currentPage?.elements.find(
+      ({ isBackground }) => isBackground
+    );
```

Comment:
Let's move this to the `useStory` selector so that it already returns the `backgroundElementId` directly and not just the whole `currentPage` object

Reply:
https://github.com/GoogleForCreators/web-stories-wp/pull/11939/commits/5beefd2fd0cd2be00e0c81c05f23fdf2579f4a8a